types/mathjs/index.d.ts

```diff
@@ -543,7 +543,7 @@ declare namespace math { // tslint:disable-line strict-export-declare-modifiers
 		/**
 		 * Create a number or convert a string, boolean, or unit to a number. When value is a matrix, all elements will be converted to number.
 		 */
-		number(value?: string|number|boolean|MathArray|Matrix|Unit|BigNumber): number|MathArray|Matrix;
+		number(value?: string|number|boolean|MathArray|Matrix|Unit|BigNumber|Fraction): number|MathArray|Matrix;
```

Comment:
I wonder if we could just make this `string | MathType`? I can't tell if this is just an enumeration of all of the values `MathType` encompasses.

Reply:
They seem different to me.

Reply:
Thank you for reviewing!